tests/test_jump.py

```diff
@@ -561,7 +567,7 @@ def test_flag_persist_groups():
         sat_expand=1.1,
         mask_persist_grps_next_int=True,
         persist_grps_flagged=0)
-
+    fits.writeto("../../../notebooks/gdqout.fits", gdqout, overwrite=True)
```

Comment:
Tests should only write pipeline files.  And all files should be written to the `pytest` directory.

Reply:
I removed the write but it is in a test that is set to skip. Removing these tests means I have to rewrite the tests every time I want to test without having to run the full step.
If there's a better way to do this, I open to suggestions.